jdaviz/configs/cubeviz/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -542,7 +545,9 @@ def _preview_x_from_extracted(self, extracted):
         return extracted.spectral_axis.value
 
     def _preview_y_from_extracted(self, extracted):
-        return extracted.flux.value
+        # TODO: use extracted's PIXAR_SR instead (but for some reason isn't populated here...)
```

Comment:
does anyone know why this is (I'm guessing its just populated later when actually loading into the app)?  Ultimately, I think the extracted spectrum just gets a copy of PIXAR_SR from the input cube, so this shouldn't matter, it just would read more cleanly if it did.